src/domain/actuator.py

```diff
@@ -1,27 +1,36 @@
 """Actuator module"""
 
-import logging
 from abc import abstractmethod
+import datetime
 from enum import Enum
+import logging
+from queue import Empty, Queue
 
 logger = logging.getLogger(__name__)
 
-# List of Actuators
-actuators = []
-
 
 class Actuator:
     """Base class of an actuator."""
 
-    class ActuatorTypes(Enum):
-        Semaphore = "Semaphore"
-        Feeder = "Feeder"
+    actuators_pool = {}
 
-    def __init__(self, id, enabled=False):
-        self.type = None
-        self.id = id
+    def __init__(self, actuator_id, enabled=False):
+        self.cmd_queue = Queue()
+        self.actuator_id = actuator_id
+        self.last_update = None
         self.enabled = enabled
         self.stop_thread = False
+        Actuator.actuators_pool[self.actuator_id] = self
+
+    def send_command(self, msg: Enum):
+        self.cmd_queue.put(msg.value)
+
+    def get_command(self):
+        self.last_update = datetime.datetime.now()
```

Comment:
check get_timestamp() function in utils.py

Reply:
get_timestamp() returns a str. Having a datetime object as last_update might be useful for further comparisons (e.g. How much time since the last ping?).